src/molecules/side-menu.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from './side-menu.module.css';
+
+const SideMenu = ({ isOpen, closeMenu, subjects }) => {
+  return (
+    <>
+      {isOpen && <div className={styles.overlay} onClick={closeMenu}></div>}
+      <div className={styles.sideMenu} data-open={isOpen ? 'true' : 'false'}>
```

Comment:
```suggestion
      <div className={styles.sideMenu} data-open={isOpen}>
```
`true` や `false` はそのまま文字列になるので, これだけでよいです

Reply:
他の修正で巻き込んだ影響だと思いますが, 修正した内容が巻き戻っているようです